### DISCUSSION TITLE
What is the quota limit for this?

### DISCUSSION THREAD

**Discussion by AustinGitHub** [sigma67/ytmusicapi#467] [Q&A] [answered]
I am looking to do this in mass for specific artists when it comes to getting their full album song data. What is the limit that this can be tested against? I want to make sure I understand before I fully start it and not get my youtube channel banned or something. Is this similar to the API quota that I have my google account under? I have youtube API quota of like 25m, is this using that in order to make these calls or is there something else behind the scenes? 

My script will call in the following 
`get_artist`
`get_artist_albums` 
`get_album`
`get_watch_playlist`

With calling all of this for 1 artist and 1 album it seems to do fine but if I wanted to do this in bulk for getting all the albums from an artist and then using the return data to then get all the songs from the albums and then checking if the songs have a music video or not and then pushing this data to my database. 

I currently due this sort of another way right now using a form of webscraping a genius page and then using youtube API to do a song search and I get the top result. Then I run another scraping method in order to try and verify the song if it belongs to specific channel name and if song title matches to the video title using php/jquery using `simple_html_dom` . This works for about 75% of the time and the rest of the songs I have to verify manually. But if I can use `ytmusicapi`, I can relieve a lot of the heavy work I need to do and make the process a lot faster. 

Hi @sigma67 can you help out with above? If more info is needed then I can do so. Thanks!

**Comment by AustinGitHub** [comment]
Example, 
Was able to get it to work for 1 album. 
![image](https://github.com/sigma67/ytmusicapi/assets/10100397/86e7685a-3085-4a7b-8cfa-6195e17aeb4b)

If I were to do this in mass, would it work? I can put this on a cron job if I know the cooldown time. Thanks!

**Comment by sigma67** [comment]
There is an open PR that tested quotas:

https://github.com/sigma67/ytmusicapi/pull/400

**Reply by sigma67** [reply, marked as answer]
FWIW I can do hundreds of searches no issue

**Reply by AustinGitHub** [reply]
Thanks!
